contentcuration/contentcuration/frontend/shared/views/MarkdownEditor/plugins/formulas/MarkdownFormulaField.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+
+  <span :class="{editing: editing}">
+    <span ref="formula"></span>
+  </span>
+
+</template>
+
+<script>
+
+  import Vue from 'vue';
+
+  import register from '../registerCustomMarkdownField.js';
+  import '../../mathquill/mathquill.js';
+
+  // vue-custom-element can't use SFC styles, so we load our styles directly,
+  // to be passed in when we register this component as a custom element
+  import css from '!css-loader!less-loader!./style.less';
+
+  const MarkdownFormulaField = {
+    name: 'MarkdownFormulaField',
+    props: {
+      editing: {
+        type: Boolean,
+      },
+      markdown: {
+        type: String,
+      },
+    },
+    data() {
+      return {
+        mathquill: null,
+      };
+    },
+    mounted() {
+      this.renderMath();
+    },
+    updated() {
+      this.renderMath();
+    },
+    methods: {
+      renderMath() {
+        this.$refs.formula.innerHTML = this.markdown;
+        this.mathquill = MathQuill.getInterface(2).StaticMath(this.$refs.formula);
```

Comment:
Yes, I think this probably got all lumped together pulled the rendering logic out of `mount`.

Reply:
Okay, I looked into this and I think it's unnecessary to even store `this.mathquill`.  I'm not even sure if the `reflow` call is necessary anymore to be honest.  It was necessary in the past when the markdown was inserted into a slot, but since the interface changed, I think I should be able to simplify this quite a bit, since the custom element should already be fully initialized by the time `renderMath` is ever called.